replybot/lib/typewheels/machine.js

```diff
@@ -150,8 +150,14 @@ function tokenWrap(state, nxt, output) {
 }
 
 
+function exec (state, nxt, surveyMetadata) {
+  if (surveyMetadata && surveyMetadata.off_date < nxt.timestamp) {
```

Comment:
Wait, this doesn't make sense to me, we shouldn't respond if the message is a MACHINE_REPORT or a PLATFORM_RESPONSE or an EXTERNAL_EVENT or anything like that. 

We can transition to an OFF state, but responding and saying the survey is off should only happen when they write us!

We can, alternatively, make an external_event which is an OFF event, which Dean manages (like timeouts) and then we just react to that event moving everyone into the correct state without having to wait on something else to happen. Maybe that's why it doesn't feel right to find the place now.